bezier-rs/docs/interactive-docs/src/components/BezierDrawing.ts

```diff
@@ -102,13 +101,13 @@ class BezierDrawing {
 
 	deselectPointHandler(): void {
 		if (this.dragIndex !== undefined) {
-			this.clearFigure();
 			this.dragIndex = null;
 			this.updateBezier();
 		}
 	}
 
-	updateBezier(options: Record<string, number> = {}): void {
+	updateBezier(options: Record<string, number> = {}, mouseLocation: Point | null = null): void {
```

Comment:
`mouseLocation?: Point` is probably more idiomatic. If its usage at the bottom of the function in `this.callback(..., mouseLocation);` requires `null` specifically, instead of any falsy value, you can do `this.callback(..., mouseLocation || null);`.